src/interface/api_interfaces/openapi.d.ts

```diff
@@ -0,0 +1,1698 @@
+/**
```

Comment:
you shouldn't need to regenerate data types if they are apart of fspiop/thirdparty.

Reply:
Ah ok, yeah I don't think we are referring to this file, so can probably just delete it, and remove it from the build task.